Samples/macOS-Swift/macOS-Swift/AppDelegate.swift

```diff
@@ -1,46 +1,10 @@
 import Cocoa
 import Sentry
+import SentrySampleShared_Swift
 
 @NSApplicationMain
 class AppDelegate: NSObject, NSApplicationDelegate {
-
     func applicationDidFinishLaunching(_ aNotification: Notification) {
-
-        // Insert code here to initialize your application
-        SentrySDK.start { options in
-            options.dsn = "https://6cc9bae94def43cab8444a99e0031c28@o447951.ingest.sentry.io/5428557"
-            options.debug = true
-            options.tracesSampleRate = 1.0
-            
-            let args = ProcessInfo.processInfo.arguments
-            
-            if args.contains("--io.sentry.profiling.enable") {
-                options.profilesSampleRate = 1
-            }
-            if #available(macOS 12.0, *), !args.contains("--disable-metrickit-integration") {
-                options.enableMetricKit = true
-                options.enableMetricKitRawPayload = true
-            }
-            
-            options.initialScope = { scope in
-                if let path = Bundle.main.path(forResource: "Tongariro", ofType: "jpg") {
-                    scope.addAttachment(Attachment(path: path, filename: "Tongariro.jpg", contentType: "image/jpeg"))
-                }
-                
-                let data = Data("hello".utf8)
-                scope.addAttachment(Attachment(data: data, filename: "log.txt"))
-                
-                scope.injectGitInformation()
-                
-                return scope
-            }
-
-            options.experimental.enableFileManagerSwizzling = !args.contains("--disable-filemanager-swizzling")
```

Comment:
This launch arg introduced in https://github.com/getsentry/sentry-cocoa/pull/4634 wasn't actually configured in any of our shared schemas, I shared it in the upstream PR at https://github.com/getsentry/sentry-cocoa/pull/5076/files#diff-384a7a6c421086af40fa69500d57db0f5122e16c6c2753bf6bbf60447e2a3c06R105 so now all sample apps can use it